### ISSUE TITLE
Staging Review finding: Confusing heading structure for information about urgent communication needs

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104797]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User can determine next steps in a flow.](https://design.va.gov/about/experience-standards/#user-can-determine-next-steps-in-a-flow)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/ask-va/introduction](https://staging.va.gov/contact-us/ask-va/introduction), there's a section of content focused on what to do if you are in the middle of a crisis. But the heading structure around this information is confusing and doesn't really describe the section of content it labels.

Currently it's coded as:
- H2: Only use Ask VA for non-urgent needs
   - H3: If you need help now, use one of these urgen communication options
       - [content about the Veterans Crisis Line and 911]

Two big concerns here:
- If I'm a user looking for information about what to do in a crisis, I might assume that "Only use Ask VA for non-urgent needs" will be a section of content about the appropriate non-urgent uses of Ask VA.
- There's only one section of content here, so putting it in an H3 subsection beneath the H2 doesn't provide a lot of value to the user.

Both of these will disproportionately impact screen reader users, who may not get the "at a glance" visual cues about this section of content. Many screen reader users navigate a page skipping heading to heading listening for something that sounds like what they need, and use the announced heading levels to understand how the content is structured. Poor headings and heading hierarchies can result in confusion and users missing critical information.

<details><summary>Link, screenshot or steps to recreate</summary>

![An H2 heading that reads "Only use Ask VA for non-urgent needs," followed by an H3 that reads "If you need help now, use one of these urgen communication option" with instructions for what to do for urgent needs.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/62304-Brian-1.png?raw=true)

</details>

## Recommended action

I would recommend a simpler structure, bumping up the H3 to an H2:

- H2: If you need help now, use one of these urgent communication options
   - [content]

If needed, the instruction to only use Ask VA for non-urgent needs should be included in the content, not in the headings.

I would also recommend you look at how similar text is structured elsewhere on VA.gov, especially in the MHV Secure Messaging service.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
